### PR TITLE
Skill calculators

### PR DESCRIPTION
**Pull request by DMills94**
TODO

- [x] Add tier info into Redux as it's used in multiply places and not a bad thing to do i think?
- [x] Add Exp multiplier override buttons to settings panel
- [ ] Add relic effects, primarily Unchained Talent
- [x] Fix bug where reset doesn't change skill in the Select but does update table back to agility, probably shouldn't change the skill just reset XP values back to either the Hiscores cache (if present) or initial state (current)
- [x] Update hiscores button
- [x] NaN values where amounts not provided in data
- [ ] Query param for specific calculator

### CODE REVIEW

**Comment by chaiinchomp** [comment]
I pulled this branch to play around with it a bit, couple of overall comments before I dig into the code:
- Do you think we should add a button to reload hiscores to the calc page?
- Seeing a few `NaN` in the inputs column (construction for example). Iirc there's a few inputs that don't have an amount since they're variable, will need to handle that case
- I think the 'Reset calculator' button is a little confusing 🤔 I thought it would set me back to my hiscores values, not 0 xp
- Agree about adding tier to redux, it's being used a lot of places now. Can store the calculated value so we aren't recomputing it all the time
- Maybe a to-do for later: add query param for linking to a specific calculator?

overall this is a great start though! Looks really nice and I like having the dropdown for switching to different skills. Nice work

File: src/components/CalculatorTable.js [hunk 1]
```diff
@@ -0,0 +1,135 @@
+/* eslint-disable no-console */
```
Comment (chaiinchomp) [inline src/components/CalculatorTable.js:1]: 🚮

File: src/store/calculators/calculators.js [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+            state.expValues[type] = {
+                ...state.expValues[type],
+                xp,
+                level,
+            };
+        },
+        updateCalculatorsExpValues: (state, action) => {
+            const { start, target } = action.payload;
+            if (!start || !target) {
+                throw new Error('You need to provide both start and target XP value objects 😠');
```
Comment (chaiinchomp) [inline src/store/calculators/calculators.js:44]: heheh, angy

**Review by chaiinchomp — COMMENTED** [review]
actually no major code comments, loooks good 👍

**Comment by DMills94** [comment]
Idk why there's so many file changes 🤔  going to recreate

**Pull request closed without merging**